src/NServiceBus.Transport.Msmq/Scripts/NServiceBus.Transport.Msmq.targets

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)\Scripts\*.ps1">
+      <Link>Scripts\%(RecursiveDir)%(Filename)%(Extension)</Link>
```

Comment:
Probably here you want **Deployment/Scripts** or **Deployment/Msmq** @indualagarsamy? This is the location the scripts end when building a project referencing the nuget.

Reply:
yep.

Reply:
I like `Deployment/Scripts` but @andreasohlund mentioned `Deployment/Msmq` in the past. I'll go with whatever you pick. :)